packages/fxa-content-server/app/tests/spec/views/connect_another_device.js

```diff
@@ -120,37 +120,79 @@ describe('views/connect_another_device', () => {
       });
     });
 
-    ['preferences', 'synced-tabs', 'tabs-sidebar', 'fxa_app_menu'].forEach(
-      (entrypoint) => {
-        const isAppMenu = entrypoint === 'fxa_app_menu';
-        describe(`with known entrypoint ${entrypoint}${
-          isAppMenu && ' and action=email'
-        }`, () => {
-          beforeEach(() => {
-            sinon.stub(view, '_isSignedIn').callsFake(() => true);
-            relier.set('entrypoint', entrypoint);
-            relier.set('context', 'fx_desktop_v3');
-            // `fxa_app_menu` without `action=email` redirects. If present, it does not redirect.
-            if (isAppMenu) {
-              relier.set('action', 'email');
-            }
-            sinon.spy(view, 'navigate');
-
-            windowMock.navigator.userAgent =
-              'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:55.0) Gecko/20100101 Firefox/55.0';
-
-            return view.render().then(() => {
-              view.afterVisible();
-            });
-          });
+    [
```

Comment:
Should all 7 of these entry points be checked? https://github.com/mozilla/fxa/blob/main/packages/fxa-content-server/app/scripts/lib/constants.js#L168

BTW, I checked them manually. The entry point param is propagated through correctly in all cases. When entrypoint=fxa_discoverability_native, the code immediately redirects to /pair/unsupported view however.

Reply:
> Should all 7 of these entry points be checked?

I'm not sure.  I'm not sure how the previous four was picked

> When entrypoint=fxa_discoverability_native, the code immediately redirects to /pair/unsupported view however.

That sounds wrong.  But when I tried it by selecting "Connect another device..." menu item on the Firefox FxA menu, it worked just fine.